config/database.yml

```diff
@@ -21,6 +21,8 @@ default: &default
   # For details on connection pooling, see rails configuration guide
   # http://guides.rubyonrails.org/configuring.html#database-pooling
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  username: postgres
+  password: io
```

Comment:
I'm not a huge fan of this because postgres defaults the username to the user rails runs as, which is me! and so my postgresql setup uses system authentication which means I never need to put a username or password in my development `database.yml` file. #spoiled 

To avoid having this file dirty in my working directory I'll have to update my `postgres` user password to match this value. 

Ideally, as a developer we should never have to modify this file on checkout to do development. So what ever gets us to that works for me, and if I have to create a user with a known password that's fine. Maybe a better solution would be to use `dotenv` where everyone can set their own username/passwords in a `.env` file that never gets checked in and those can be pulled in here like `<%= ENV.fetch('POSTGRES_USERNAME') %>`. This type of change sounds like a new issue.